deploy.sh

```diff
@@ -3,15 +3,26 @@
 push_helm_charts() {
 	PACKAGE=$1
 	CHANGED=0
-	VERSION=$(grep 'version:' deploy/"$PACKAGE"/helm/Chart.yaml | awk '{print $2}')
+	VERSION=${TRAVIS_TAG//v}  # Set version (without v prefix)
```

Comment:
am not aware how TRAVIS_TAG works, can you just make use of `ENV_CSI_IMAGE_VERSION` in deploy.sh?

Reply:
The `TRAVIS_TAG` is set to the git tag else it is empty:
https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

Will look into this tho, it seems that the scripts exists when it is a tagged build. TRAVIS_BRANCH is then set to TRAVIS_TAG value and will en dup in the else and exit. Is this expected?

Reply:
we update the csi container image (canary) once each PR is merged. in the same way we need to update the helm chat for the master branch. 

will this work like it?

I suggest let's use `ENV_CSI_IMAGE_VERSION` in this PR, will move to use TRAVIS_TAG later when we integrate Travis with CD